apps/service_providers/forms.py

```diff
@@ -152,6 +152,14 @@ class TurnIOMessagingConfigForm(ObfuscatingMixin, ProviderTypeConfigForm):
     auth_token = forms.CharField(label=_("Auth Token"))
 
 
+class SureAdhereMessagingConfigForm(ObfuscatingMixin, ProviderTypeConfigForm):
+    obfuscate_fields = ["client_secret"]
+
+    client_id = forms.CharField(label=_("Client ID"))
+    client_secret = forms.CharField(label=_("Client Secret"))
+    base_url = forms.CharField(label=_("Base URL"))
```

Comment:
This should use a URL field:
```suggestion
    base_url = forms.URLField(label=_("Base URL"), validators=[URLValidator(schemes=["https"]))
```